ApiTests/Tests/UI/IFrameTests.cs

```diff
@@ -1,30 +1,45 @@
 ﻿using ApiTests.Tests.UI;
 using OpenQA.Selenium;
+using static System.Net.Mime.MediaTypeNames;
 
 namespace TestProject.Tests.UI
 {
     internal class IFrameTests : BaseTest
     {
-        private static readonly By framesBtn = By.XPath(string.Format(preciseTextXpath, "Frames"));
-        private static readonly By iframeBtn = By.XPath(string.Format(preciseTextXpath, "iFrame"));
-        //private static readonly By editBtn = By.XPath(insert locator here);
+        private static readonly By framesBtn = By.XPath(string.Format("//*[@href=\"/frames\"]", "Frames"));
+        private static readonly By iframeBtn = By.XPath(string.Format("//a[@href=\"/iframe\"]", "iFrame"));
+        private static readonly By undoBtn = By.XPath(string.Format("//div[@class=\"tox-collection__item-label\"])[1]", "textField"));
+        private static readonly By editBtn = By.XPath(string.Format("//button[@class=\"tox-mbtn tox-mbtn--select\"])[2]", "editBtn"));
+        private static readonly By textField=By.Id(string.Format("id=\"tinymce\"","textField"));
         //private static readonly By undoBtn = By.XPath(insert locator here);
         private static readonly string randomValue = Guid.NewGuid().ToString();
-        private static readonly string initText = "Your content goes here.";
+        private static readonly string initText =  "Your content goes here.";
 
         [Test]
         public void IFrameTest()
         {
+            IFrameTests frameTests = new IFrameTests();
+            Console.WriteLine(initText);
```

Comment:
In most of the cases console output won't be of use, as usually auto tests are executed on a remote machine and we'll have info only from assertions / logs